ast/builtins.go

```diff
@@ -1425,18 +1426,27 @@ var JSONPatch = &Builtin{
 	Categories: objectCat,
 }
 
-var ObjectGet = &Builtin{
-	Name: "object.get",
-	Description: "Returns value of an object's key if present, otherwise a default. " +
-		"If the supplied `key` is an `array`, then `object.get` will search through a nested object or array using each key in turn. " +
-		"For example: `object.get({\"a\": [{ \"b\": true }]}, [\"a\", 0, \"b\"], false)` results in `true`.",
+var ObjectSubset = &Builtin{
+	Name:        "object.subset",
+	Description: "Determines if an object `sub` is a subset of another object `super`. Object `sub` is a subset of object `super` if and only if every key in `sub` is also in `super`, **and** for all keys which `sub` and `super` share, they have the same value. This function works with objects, sets, and arrays. If both arguments are objects, then the operation is recursive, e.g. `{\"c\": {\"x\": {10, 15, 20}}` is a subset of `{\"a\": \"b\", \"c\": {\"x\": {10, 15, 20, 25}, \"y\": \"z\"}`. If both arguments are sets, then this function checks if every element of `sub` is a member of `super`, but does not attempt to recurse. If both arguments are arrays, then this function checks if `sub` appears contiguously in order within `super`, and also does not attempt to recurse.",
```

Comment:
[nit] This is a super-long string. In over places, I've used `"..." + "..."` to break it into pieces.

Reply:
75c1493bfe28dd040b4636ae11a8a78e99159eea